docs/builtins.md

```diff
@@ -0,0 +1,225 @@
+# Builtins
+
+Evy provides built-in functions and events that allow for user
+interaction, graphics, animation, mathematical operations, and more.
+
+Functions are self-contained blocks of code that perform a specific
+task. Events are notifications that are sent to a program when
+something happens, such as when a user moves the mouse or presses a
+key.
+
+## Input and Output
+
+### `print`
+
+`print` prints the arguments given to it to the output area. It separates them by
+a single space and outputs a newline character at the end.
+
+#### Example 
+
+```evy
+print "Hello"
+print 2 true "blue"
+print "array:" [1 2 3]
+print "map:" {name:"Scholl" age:21}
+```
+Output
+```evy:output
+Hello
+2 true blue
+array: [1 2 3]
+map: {name:Scholl age:21}
+```
+
+#### Reference 
+
+    print a:any...
+
+The `print` function prints its arguments to the output area, each
+separated by a single space and terminated by a newline character. If
+no arguments are provided it only prints the newline character. 
+
+A tab is printed for the `\t` control character. A newline is printed
```

Comment:
technically the tab is the "control character". `\t` is an escape sequence or backslash sequence or something like that

Reply:
ok. I have reworded the whole thing a little, hope that's ok.